### PR TITLE
added translation script

### PR DESCRIPTION
**Pull request by owos**

### CODE REVIEW

**Comment by ToluClassics** [comment]
Thanks! can you add dependencies to `requirements.txt`

File: tranlation_script/translate_queries.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        type=str,
+        required=True,
+        help="original language of the data to be translated",
+        choices=TARGET_LANG,
+    )
+    parser.add_argument(
+        "--pivot",
+        type=str,
+        required=True,
+        default="en",
```
Comment (ToluClassics) [inline tranlation_script/translate_queries.py:48]: can we restrict options here to "fr" and "en"

**Comment by ToluClassics** [comment]
https://github.com/ToluClassics/masakhane_xqa/issues/3

File: tranlation_script/translate_queries.py (hunk 1 above)
Comment (owos) [reply tranlation_script/translate_queries.py:48]: Yes. I'll do that now.

**Review by ToluClassics — APPROVED** [review]

**Pull request merged**